iree/compiler/Dialect/HAL/Target/LLVM/LLVMIRPasses.cpp

```diff
@@ -66,6 +66,8 @@ std::unique_ptr<llvm::TargetMachine> createTargetMachine(
 LogicalResult runLLVMIRPasses(const LLVMTargetOptions &options,
                               llvm::TargetMachine *machine,
                               llvm::Module *module) {
+  if(options.addressSanitizer)
```

Comment:
style: (almost) always use braces for multi-line statements in IREE code:
https://google.github.io/styleguide/cppguide.html#Conditionals